src/services/resolve/services/userResolveService.js

```diff
@@ -0,0 +1,103 @@
+const { authenticate } = require('@feathersjs/authentication');
+const errors = require('@feathersjs/errors');
+
+// get an json api conform entry
+const getDataEntry = ({
+	type, id, name, authorities = ['can-read'], attributes = {},
+}) => ({
+	type,
+	id,
+	attributes: Object.assign({}, {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an object spread instead of `Object.assign` eg: `{ ...foo }`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4977296)